About.html

```diff
@@ -0,0 +1,115 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Music Concert</title>
+    <link rel="stylesheet" type="text/css" href="about.css">
+    <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css">
+    <script src="https://kit.fontawesome.com/0f56b98a6f.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header>
+        <div class="header1">
+            <i class="fa-brands fa-facebook-f"></i>
+            <i class="fa-brands fa-twitter"></i>
+            <a href="#">English</a>
+            <a href="#">My Page</a>
+            <a href="#" class="logout">Logout</a>
+        </div>
+        <div class="header2">
+        <div class="header-logo">
+            <img src="Images/white logo.png">
+        </div>
+        <div>
+            <button class="hambuger">
+                <i class="fa-solid fa-bars"></i>
+            </button>
+        </div>
+        <nav class="nav-menu">
+            <button class="close">
+                <i class="fa-solid fa-xmark"></i>
+            </button>
+            <ul>
+                    <li><a href="index.html" class="item-about">Home</a></li>
+                    <li><a href="#" class="item-program">Program</a></li>
+                    <li><a href="#" class="item-join">Join</a></li>
+                    <li><a href="#" class="item-sponsor">Sponsor</a></li>
+                    <li><a href="#" class="item-newa">News</a></li>
+                   
+                </ul>
+                </nav>
+            <button class="campaign">
+                Referral program
+            </button>
+        </div>
+    </header>
+    <section class="about">
+        <div class="small-about">
+        <h3>"Join the Musical Celebration, Experience Unity!"</h3>
+        <h1>HARMONY IN MOTION 2023</h1>
+        <p class="description">Experience the magic of music like never before with 'Harmony in Motion 2023'. This incredible event is set to captivate your senses and transport you to a world where rhythm and melody reign supreme. Whether you're a fan of soulful ballads or high-energy beats, our talented artists will take you on a musical journey that you won't forget. With stunning visuals, electrifying performances, and an ambiance that exudes harmony, this is one event that you simply cannot miss. So, mark your calendars and get ready to immerse yourself in the sound of 'Harmony in Motion 2023'!</p>
+        <p class="date">
+            We are more than happy to assist you in any way we can. Thank you for your interest in "Harmony in Motion 2023" and we hope to see you there! For any further questions, please feel free to contact us at <br><a href="#">harmonyinmotion2023@gmail.com</a>.</p>
+        </div>
+        </section>
+    <section class="logo">
+        <h3>Harmony In Motion 2023</h3>
+        <hr>
+        <p></p>
+        <img src="Images/Harmony in motion logo.png">
+    </section>
```

Comment:
- [ ] At the moment, the size of the logo image does not match that of the original design. Please kindly ensure that it matches the original design
- [ ] Also, kindly include a text after the section header as it is in the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/219871513-42cd4d4f-0d3c-4074-9ed2-8a6fee82d824.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/219871553-022d68d9-642e-4e36-b180-00aa0d4c9b93.png)